src/main/java/org/openmrs/contrib/dwr/struts/StrutsCreator.java

```diff
@@ -1,178 +0,0 @@
-/*
- * Copyright 2005 Joe Walker
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.openmrs.contrib.dwr.struts;
-
-import java.lang.reflect.Method;
-
-import javax.servlet.ServletContext;
-import javax.servlet.http.HttpServletRequest;
-
-import org.apache.struts.action.ActionForm;
-import org.apache.struts.config.ModuleConfig;
-import org.apache.struts.util.RequestUtils;
-import org.openmrs.contrib.dwr.WebContext;
-import org.openmrs.contrib.dwr.WebContextFactory;
-import org.openmrs.contrib.dwr.create.AbstractCreator;
-import org.openmrs.contrib.dwr.extend.Creator;
-import org.openmrs.contrib.dwr.util.FakeHttpServletRequest;
-import org.openmrs.contrib.dwr.util.LocalUtil;
-import org.openmrs.contrib.dwr.util.Logger;
-import org.openmrs.contrib.dwr.util.Messages;
-
-/**
- * StrutsCreator
- * @author Ariel O. Falduto
- * @author Joe Walker [joe at getahead dot ltd dot uk]
- */
-public class StrutsCreator extends AbstractCreator implements Creator
```

Comment:
Did you intentionally delete this?

Reply:
Yeah, migrating these will take some extra effort, and I don't think they're used anywhere in the Legacy UI.